includes/class-wc-calypso-bridge-base-admin-menu.php

```diff
@@ -0,0 +1,396 @@
+<?php
+
+use Automattic\Jetpack\Status;
+
+/**
+ * Base Admin Menu file.
+ *
+ * @since   x.x.x
+ * @version x.x.x
+ *
+ * The base admin menu controller for Ecommerce WoA sites.
```

Comment:
Would be good to document/explain why we've copied this file from Jetpack. Maybe link to the source file.

Reply:
Added a link to the Jetpack source file.